bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
```

Comment:
`[\s*|checkmark-character]` --> `(?:\s|checkmark-character)*`

`[]` denotes a character class and is not the same as a group `()`. Also you don't need the quantifier behind `\s` if you define it for the group :+1: